string-manipulation-3/src/is-palindromic.js

```diff
@@ -0,0 +1,9 @@
+/* exported isPalindromic */
+function isPalindromic(string) {
+  var stringSplit = string.split(' ').join('');
+  var reversedString = string.split('').reverse().join('').split(' ').join('');
+  if (stringSplit === reversedString) {
+    return true;
+  }
+  return false;
```

Comment:
If you are just returning true or false you can just return the comparison, it will evaluate to `true` or `false` then that value will be returned.

Example:
```JAVASCRIPT
return stringSplit === reversedString;
```